packages/client/scripts/getCoilSettings.ts

```diff
@@ -42,11 +42,17 @@ export const getCoilSetting = async () => {
 
   const coilSetting = await knexClient
     .select()
-    .from<CoilSettingType>(coilSettingPath)
-    .then(([dbCoil]) => {
-      if (dbCoil) {
-        return dbCoil
+    .from<EngineSettingType>(engineSettingPath)
```

Comment:
We need to update this file to be getEngineSetting and them make it get only those keys which are required.

Reply:
@hanzlamateen done.